tools/docker/ubuntu-tdx/Dockerfile

```diff
@@ -0,0 +1,161 @@
+# SPDX-License-Identifier: MPL-2.0
+
+FROM intelcczoo/tdvm:ubuntu22.04-mvp_2023ww15 as build-base
```

Comment:
What are the differences between the two Dockerfile under `ubuntu` and `ubuntu-tdx`? Is it possible to write a template-based `gen_dockerfile.sh` like we have done for OSDK?

Reply:
In the TDX version, we don't need to build QEMU, GRUB, OVMF in the Dockerfile. I will try to use some scripts to control whether to compile these parts.